### PR TITLE
Add Epoch.flix

### LINKED ISSUE #2110: Add Epoch
Add the following `Epoch.flix`:

```
namespace Epoch {

  /// 
  /// Returns the number of nano seconds since the Unix epoch.
  /// 
  pub def nanoSeconds(): Int64 & Impure

  /// 
  /// Returns the number of micro seconds since the Unix epoch.
  /// 
  pub def microSeconds(): Int64 & Impure

  /// 
  /// Returns the number of mili seconds since the Unix epoch.
  /// 
  pub def miliSeconds(): Int64 & Impure

  /// 
  /// Returns the number of seconds since the Unix epoch.
  /// 
  pub def seconds(): Int64 & Impure

}
```

This is intended to compliment `Environment.flix`. The idea is that these can be used for quick and dirty benchmarking.

Note as for `Environment` namespaces may be reorganized at a later point.

No tests are necessary. Just do a few manual tests to see that it looks correct!

### PR DESCRIPTION
**Pull request by jaschdoc**
Resolves #2110

### CODE REVIEW

File: main/src/library/Epoch.flix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    ///
+    /// Returns the number of micro seconds since the Unix epoch.
+    ///
+    pub def microSeconds(): Int64 & Impure = ???
+
+    ///
+    /// Returns the number of mili seconds since the Unix epoch.
+    ///
+    pub def miliSeconds(): Int64 & Impure = ???
```
Comment (mlutze) [inline main/src/library/Epoch.flix:16]: should be `milliseconds` if [Wiktionary](https://en.wiktionary.org/wiki/millisecond) is to be trusted.

Comment (magnus-madsen) [reply main/src/library/Epoch.flix:16]: same for nanoseconds etc?

Comment (mlutze) [reply main/src/library/Epoch.flix:16]: I was referring to the double `l`. But all-lowercase for all of them feels more natural to me.

Comment (magnus-madsen) [reply main/src/library/Epoch.flix]: Doh,ok

Comment (jaschdoc) [reply main/src/library/Epoch.flix]: Ok changing it to lower case and double l :)

**Comment by magnus-madsen** [comment]
Thanks!

**Pull request merged**